end-to-end-test/local/specs/core/studyview.screenshot.spec.js

```diff
@@ -54,3 +56,87 @@ describe('study view generic assay categorical/binary features', function() {
         assertScreenShotMatch(res);
     });
 });
+
+describe('study view x vs y charts', () => {
+    const X_VS_Y_CHART = `div[data-test="chart-container-X-VS-Y-AGE-MUTATION_COUNT"]`;
+    const X_VS_Y_HAMBURGER_ICON = `${X_VS_Y_CHART} [data-test="chart-header-hamburger-icon"]`;
+    const X_VS_Y_MENU = `${X_VS_Y_CHART} [data-test="chart-header-hamburger-icon-menu"]`;
+    before(() => {
+        const url = `${CBIOPORTAL_URL}/study?id=lgg_ucsf_2014_test_generic_assay`;
+        goToUrlAndSetLocalStorage(url, true);
+        waitForNetworkQuiet();
+
+        // remove mutation count vs diagnosis age chart if it exists
+        if ($(X_VS_Y_CHART).isExisting()) {
+            jsApiHover(X_VS_Y_CHART);
+            $(X_VS_Y_CHART)
+                .$('.deleteChart')
+                .waitForDisplayed();
+            $(X_VS_Y_CHART)
+                .$('.deleteChart')
+                .click();
+            browser.waitUntil(() => {
+                return !$(X_VS_Y_CHART).isExisting();
+            });
+        }
+    });
+    it('adds mutation count vs diagnosis age chart', () => {
+        $(ADD_CHART_BUTTON).waitForDisplayed({
+            timeout: WAIT_FOR_VISIBLE_TIMEOUT,
+        });
+        $(ADD_CHART_BUTTON).click();
+
+        waitForNetworkQuiet();
+
+        // Change to X vs Y tab
+        $(ADD_CHART_X_VS_Y_TAB).waitForDisplayed({
+            timeout: WAIT_FOR_VISIBLE_TIMEOUT,
+        });
+        $(ADD_CHART_X_VS_Y_TAB).click();
+
+        // Add Mutation Count vs Diagnosis Age chart
+        // X-axis: diagnosis age
+        $('.xvsy-x-axis-selector').waitForDisplayed();
+        $('.xvsy-x-axis-selector').click();
+        $('.xvsy-x-axis-selector #react-select-2-option-0').waitForDisplayed();
+        $('.xvsy-x-axis-selector #react-select-2-option-0').click();
```

Comment:
One question about this `#react-select-2-option-0`, will this always work? I am not so sure about how react-select assign id to components...

Reply:
it's definitely worrisome... I will look into if I can query the options by text content